### PR TITLE
VZ-5186: Sample application for VerrazzanoWebLogicWorkload

### PR DESCRIPTION
**Pull request by bhatpmk**
# Description

This PR includes a simple auxiliary image to test  VerrazzanoWebLogicWorkload.

Contributes to VZ-5186

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: tests/testdata/test-applications/weblogic/hello-verrazzano/pom.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright (c) 2022, Oracle and/or its affiliates. -->
+<!-- Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.oracle.verrazzano.weblogic.application</groupId>
```
Comment (markxnelson) [inline tests/testdata/test-applications/weblogic/hello-verrazzano/pom.xml:9]: io.verrazzano not com.oracle.verrazzano

File: tests/testdata/test-applications/weblogic/hello-verrazzano/setup/build.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+cp ../target/hello.war ./wlsdeploy/applications
+zip -r wdt_archive.zip wlsdeploy
+rm -r wlsdeploy
+
+echo 'Download WDT...'
+if [ -f weblogic-deploy.zip ]; then
+    echo 'Using existing weblogic-deploy.zip...'
+else
+    echo 'Downloading weblogic-deploy.zip...'
+    wget https://github.com/oracle/weblogic-deploy-tooling/releases/download/weblogic-deploy-tooling-1.9.0/weblogic-deploy.zip
```
Comment (markxnelson) [inline tests/testdata/test-applications/weblogic/hello-verrazzano/setup/build.sh:24]: update the version?

File: tests/testdata/test-applications/weblogic/hello-verrazzano/setup/exporter-config.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
+metricsNameSnakeCase: true
```
Comment (markxnelson) [inline tests/testdata/test-applications/weblogic/hello-verrazzano/setup/exporter-config.yaml:1]: copyright notice

Comment (markxnelson) [reply tests/testdata/test-applications/weblogic/hello-verrazzano/setup/exporter-config.yaml:1]: maybe it can't tolerate it - don't recall

**Review by markxnelson — COMMENTED** [review]
couple of small things

File: tests/testdata/test-applications/weblogic/hello-verrazzano/pom.xml (hunk 1 above)
Comment (bhatpmk) [reply tests/testdata/test-applications/weblogic/hello-verrazzano/pom.xml]: Corrected

File: tests/testdata/test-applications/weblogic/hello-verrazzano/setup/build.sh (hunk 2 above)
Comment (bhatpmk) [reply tests/testdata/test-applications/weblogic/hello-verrazzano/setup/build.sh]: Updated to the latest version

File: tests/testdata/test-applications/weblogic/hello-verrazzano/setup/exporter-config.yaml (hunk 3 above)
Comment (bhatpmk) [reply tests/testdata/test-applications/weblogic/hello-verrazzano/setup/exporter-config.yaml:4]: Added copyright. I need to check whether metricsNameSnakeCase needs to be removed. I would like to keep this for the time being.

**Review by markxnelson — APPROVED** [review]

**Pull request merged**